src/main/SettingsUpgradeUtils.cpp

```diff
@@ -133,23 +133,50 @@ getCreateTx(PublicKey const& publicKey, LedgerKey const& contractCodeLedgerKey,
 
     tx.ext.v(1);
     tx.ext.sorobanData().resources = uploadResources;
-    tx.ext.sorobanData().resourceFee = 1'000'000;
+    tx.ext.sorobanData().resourceFee = 3'000'000;
 
     return {txEnv, contractSourceRefLedgerKey, contractID};
 }
 
+void
+validateConfigUpgradeSet(ConfigUpgradeSet const& upgradeSet)
```

Comment:
Can we quickly cover this check triggering in some test?

Reply:
Done